pytest_selenium/pytest_selenium.py

```diff
@@ -43,8 +43,8 @@ def pytest_addhooks(pluginmanager):
 def session_capabilities(pytestconfig):
     """Returns combined capabilities from pytest-variables and command line"""
     driver = pytestconfig.getoption('driver').upper()
-    capabilities = getattr(DesiredCapabilities, driver, {}).copy()
-    if driver == 'REMOTE':
+    capabilities = getattr(DesiredCapabilities, driver.upper(), {}).copy()
+    if driver.upper() == 'REMOTE':
```

Comment:
driver is already converted to uppercase on line 45, so we can remove these changes

Reply:
*the feeling of shame*